portfolio/src/main/webapp/css/style.css

```diff
@@ -31,17 +48,50 @@
     color: #000;
 }
 
+.navbar a i {
+    color: #fff;
+}
+
+.navbar.scrolled a i {
+    color: #000;
+}
+
+
 nav a {
     color: #fff;
 }
 
+.text-white {
```

Comment:
Nit: I'd recommend naming this more semantically, around the type of text that you are trying to style. Should all dark-theme text be white? In that case, we can change the class name to .dark-theme, and we semantically know that all text within the dark theme should be white. This makes it easier for another developer to pick up where you left off, and have a semantic understanding of what the class and its styles do!

Check out this doc: go/htmlcssstyle#ID_and_Class_Naming